docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,22 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+Learn how to integrate with Eclipse:
+
+* [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+* [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+* [[Import an existing sbt build into Eclipse|EclipseSbt]]
+    
+Learn how to integrate with Intellij:
+
+* [[Import an existing Maven project into Intellij|intellijMaven]]
+* [[Import an existing sbt build into Intellij|IntellijSbtJava]]
+    
+Learn more about Lagom:
+
+* [[Core concepts|CoreConcepts]]
+* [[Using the service locator|ServiceLocator]]
```

Comment:
What I found a bit strange here was that the first item is `Core concepts` which covers a lot but starts off with exactly the list of links to:
 * Service API
 * Persistence API
 * Broker API <-- In a previous comment Tim and I thought this was missing here.
 * Development Environment
 * Production Environment

So if we put all comments together the list inside `Core Concepts` and the list in `What's next` (just under the `core concepts` bullet look very similar.

- Using the service locator <-- we thought we should remove this one 
- Writing Lagom service <-- we'd add this one
- Using the message broker
- Using publish-subscribe
- Using persistent entities <-- this looks quite unimportant

So the lists have almost the same elements (except for ServiceLocator+pubsub  vs   devmode+prodEnv). Hmmm, something's going on here and I don't see what it is. @rstento did you have an idea in mind that drove you into this differentiation?

Finally, I think there's few bits that should always be presented in the same order:

 * Service API
 * Persistence API
 * Broker API